### ISSUE TITLE
Date and Time component in the CMS has a visually hidden All Day checkbox field

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10131]
## Description
The Date and Time component used in the Event content type has an All Day checkbox field but it is visually hidden using absolute positioning. This causes confusion because visual users do not have access to this field but it is still available and accessible to screen readers.

## Screenshot
![image](https://user-images.githubusercontent.com/106678594/183913237-5ec6f3d8-7df8-46ee-937b-fa2f1da1233a.png)

## Accessibility Standard

WCAG version 2.0 A, [Criterion 1.3.1](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

## Acceptance Criteria
- [ ] Determine if this field should be available or not for all users
- [ ] Technical review
- [ ] Change management consulted
- [ ] Implementation ticket created

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by laflannery** [comment]
This was resolved by Public Websites in a separate issue #10324
